assets/src/edit-story/components/header/buttons/index.js

```diff
@@ -47,15 +47,20 @@ const Space = styled.div`
   width: 6px;
 `;
 
+const Spinner = styled.div`
+  position: absolute;
```

Comment:
Why is this needed / what is this now positioned to in relation?

Reply:
This will now be positioned on top of the first icon of the buttons area, either the "Save Draft" or "Switch to Draft":
<img width="395" alt="Screenshot 2021-02-11 at 13 47 51" src="https://user-images.githubusercontent.com/3294597/107632903-70830c80-6c67-11eb-963d-7cca84a04300.png">

We could add a wrapper around the first icon and the spinner together to make it more explicit. WDYT?